openvswitch.spec

```diff
@@ -714,6 +715,10 @@ fi
 %{_unitdir}/ovn-controller-vtep.service
 
 %changelog
+* Fri Oct 26 2017 Numan Siddique <nusiddiq@redhat.com> - 2.8.1
```

Comment:
changelog entry with 2.8.1-2 was ok, it's Version-Release but let's merge as-is

Reply:
it's also bogus date, I'll fixup in a followup commit
warning: bogus date in %changelog: Fri Oct 26 2017 Numan Siddique <nusiddiq@redhat.com> - 2.8.1